src/Explorer/Controls/Settings/SettingsSubComponents/SubSettingsComponent.tsx

```diff
@@ -197,17 +197,20 @@ export class SubSettingsComponent extends React.Component<SubSettingsComponentPr
           </MessageBar>
         )}
         {this.props.timeToLive === TtlType.On && (
-          <TextField
-            id="timeToLiveSeconds"
-            styles={getTextFieldStyles(this.props.timeToLiveSeconds, this.props.timeToLiveSecondsBaseline)}
-            type="number"
-            required
-            min={1}
-            max={Int32.Max}
-            value={this.props.timeToLiveSeconds?.toString()}
-            onChange={this.onTimeToLiveSecondsChange}
-            suffix="second(s)"
-          />
+          <div>
```

Comment:
Why do we need wrap the textfield around a div?

Reply:
It has no use here. It is a change from the previous commit, I tried using the labels. I will remove the div and make a commit